src/gmt_support.c

```diff
@@ -10323,7 +10338,7 @@ GMT_LOCAL unsigned int gmtsupport_contour_old_T_parser (struct GMT_CTRL *GMT, ch
 		}
 		else {
 			GMT_Report (GMT->parent, GMT_MSG_ERROR,
-			            "Option -T: Give low and high labels either as :LH or :<low>,<high>.\n");
+						"Option -T: Give low and high labels either as :LH or :<low>,<high>.\n");
```

Comment:
Likewise here and below.

Reply:
I did a few more but dont want to go over all of this manually.  As you know I am not in favour of 80character screens so feel these break-the-line adds longer code and more opportunities for tab/space shits.

Reply:
It's has nothing to do about the the line width but only on a good alignment with above line. Tee price of indenting with tabs is that we must do a mix of tabs and spaces to keep that alignment.

Reply:
If you want to revert some of these then no problem, go right ahead.